README.md

```diff
@@ -54,11 +55,16 @@ positional arguments:
 
 optional arguments:
   -h, --help            show this help message and exit
+  --skip                Directory (ie. env/foo) or files (ie. cool.py) to skip
+                        Arguments are seperated by comma (ie. env,kung.py,foo.py)
+  --isort               Only process with isort (imports only)
+  --black               Only process with black (code only)
```

Comment:
this names can be more descriptive to their purpose

Something like 
`--isort-only`
`--black-only`

Reply:
that's no problem, thought about it, but in the context of pyfmt what else could it mean to just have `isort` instead of `isort-only`

I'll change it